recipes/simpleaudio/recipe.yaml

```diff
@@ -0,0 +1,51 @@
+context:
+  name: simpleaudio
+  version: "1.0.4"
+  python_min: "3.6"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: 691c88649243544db717e7edf6a9831df112104e1aefb5f6038a5d071e8cf41d
+
+build:
+  script: python -m pip install . -vv
+  number: 0
+  noarch: Y
```

Comment:
i'm sorry, mean that as an example.... just remove `noarch` altogether.

Reply:
I thought that be the case